Lib/test/test_types.py

```diff
@@ -1150,22 +1194,23 @@ class N(type, metaclass=M):
 
 class SimpleNamespaceTests(unittest.TestCase):
 
-    def test_constructor(self):
-        ns1 = types.SimpleNamespace()
-        ns2 = types.SimpleNamespace(x=1, y=2)
-        ns3 = types.SimpleNamespace(**dict(x=1, y=2))
+    # TODO: RUSTPYTHON
+    # def test_constructor(self):
```

Comment:
Why is this test commented out?

Reply:
there is rust part bug to make crash and I found I can't fix it quickly.

Reply:
Thats fine but lets mark tests like that with `@unittest.skip("TODO: RUSTPYTHON")` instead of comment out. It will make them easier to find later.

Reply:
In that case, the test immediately stopped by crash so never run later tests.

Reply:
Oh, ok skip would be the correct solution